server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/BackgroundTaskItem.java

```diff
@@ -39,29 +37,29 @@ public SelenideElement getComponent() {
 
   public BackgroundTaskItem openActions() {
     elt.$(".js-task-action > .dropdown-toggle").click();
-    elt.$(".js-task-action > .dropdown-menu").shouldBe(visible);
+    elt.$(".js-task-action > .dropdown-menu").shouldBe(Condition.visible);
     return this;
   }
 
-  public BackgroundTaskItem openScannerContext () {
+  public BackgroundTaskItem openScannerContext() {
     elt.$(".js-task-show-scanner-context").click();
-    $(".js-task-scanner-context").shouldBe(visible);
+    Selenide.$(".js-task-scanner-context").shouldBe(Condition.visible);
     return this;
   }
 
   public BackgroundTaskItem assertScannerContextContains(String text) {
-    $(".js-task-scanner-context").should(hasText(text));
+    Selenide.$(".js-task-scanner-context").should(Condition.hasText(text));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "hasText"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)